bundles/org.eclipse.osgi/META-INF/MANIFEST.MF

```diff
@@ -7,10 +7,12 @@ Export-Package: org.eclipse.core.runtime.adaptor;x-friends:="org.eclipse.core.ru
   uses:="org.eclipse.osgi.report.resolution,
    org.osgi.framework.wiring,
    org.eclipse.osgi.framework.eventmgr,
+   org.apache.felix.resolver,
```

Comment:
@laeubi this uses constraint seems wrong. No public facing API in `org.eclipse.osgi.container` uses the `org.apache.felix.resolver`.  This package is only used by package private class `ModuleResolver`

Reply:
Should be fixed with next i-build tomorrow:
- https://github.com/eclipse-pde/eclipse.pde/pull/1830